docs/system/wind_turbine.md

```diff
@@ -1,7 +1,147 @@
+
+The synchronous speed $n_s$ in *rpm* is dependend on the pole count $p$ (as pairs, four poles -> two pairs)  of the stator and the excitation frequenzy $f_e$:
+
+$$n=\frac{f_e\cdot60}{p}$$
+
+The synchronous speed for a four pole generator on grid frequenzy of $f=50Hz$ is $1500rpm$. To achieve these speeds, a gearbox is installed. Alternatively the pole pairs could be increased, leading to higher overall mass. To run a turbine at $n_s=120rpm$ and $f=e=50Hz$, $p=25$ pole pairs (!) are needed which is simply not feasible for this type of machine. To manipulate the synchronous speed to allow variable turbine speed, several approaches are possible. The quasi-standard are double-fed induction generators (DFIG) (always as wound-rotor) where the field windings are fed with an adjustable frequency AC-power [4] while the armature windings are directly connected to the grid or other equipment for conversion. With this setup, the rotational speed of the turbine can vary with windspeed while the active *and* reactive power of the generator can be controlled.
+
+<figure>
+    <center>
+        <a href="https://commons.wikimedia.org/wiki/File:DFIG_in_Wind_Turbine.svg#/media/File:DFIG_in_Wind_Turbine.svg">
+            <fig-caption src="system/DFIG_in_Wind_Turbine.svg" caption="Schematics for a double fed induction generator [5]" style="width:50%"  num="3" />
+        </a>
+    </center>
+</figure>
+
+### Synchronous
+
+#### Field Coil Excitation
+
+Synchronous generators can be build with a much higher pole count than asynchronous generators and are used for gearless setups which reduces the maintance requirements significantly. If they are directly coupled to the grid, they need to be synchronized first and cannot run with different speeds. In fact, when the force on the turbine increases and the generator comes out of sync with the grid, the system can be damage due to balancing currents.
+Most turbines have a rectifier and a [power inverter](inverter) to connect to the grid.
+
+#### Permanent Magnet Excitation
+
+Exitation with permanent magnets reduces the complexity and therefor maintance. For this reason, PM generators are used mostly on offshore wind farms. There is also no energy needed for exitation so they have slightly better efficieny. Considering the large amount of rare earths needed for permanent magnets though, this effect is negligible [6].
+
+The axial flux or axial gap generators are a special type of synchronous PM generators not widely found in larger applications. It can be however found frequently for small scale DIY turbines. Their flat structure and easy-to-wind coils makes them very suited for small workshops. They provide a fairly good efficiency and common materials such as wood, steel and epoxy can be used. Since the rotor becomes wider with increasing pole count, rotational inertia and centifugal forces are increased as well, though higer pole count mostly means lower rpm needed.
```

Comment:
```suggestion
The axial flux or axial gap generators are a special type of synchronous PM generators not widely found in larger applications. It can however be found frequently for small scale DIY turbines. Their flat structure and easy-to-wind coils makes them very suited for small workshops. They provide a fairly good efficiency and common materials such as wood, steel and epoxy can be used. Since the rotor becomes wider with increasing pole count, rotational inertia and centrifugal forces are increased as well, though higher pole count mostly means lower rpm needed.
```